x/stakeibc/types/params.go

```diff
@@ -242,8 +255,103 @@ func isCommission(i interface{}) error {
 	return nil
 }
 
+func validMaxRedemptionRates(i interface{}) error {
+	rrs, ok := i.(map[string]string)
+	if !ok {
+		return fmt.Errorf("max redemption rates not accepted: %T", i)
+	}
+
+	for _, rateStr := range rrs {
+		rate, err := sdk.NewDecFromStr(rateStr)
+		if err != nil {
+			return err
+		}
+		if !rate.IsPositive() {
+			return fmt.Errorf("redemption rate should be positive")
+		}
+	}
+
+	return nil
+}
+
+func validMinRedemptionRates(i interface{}) error {
+	rrs, ok := i.(map[string]string)
+	if !ok {
+		return fmt.Errorf("min redemption rates not accepted: %T", i)
+	}
+
+	for _, rateStr := range rrs {
+		rate, err := sdk.NewDecFromStr(rateStr)
+		if err != nil {
+			return err
+		}
+		if !rate.IsPositive() {
+			return fmt.Errorf("redemption rate should be positive")
+		}
+	}
+	return nil
+}
+
 // Validate validates the set of params
 func (p Params) Validate() error {
+	if err := isPositive(p.DepositInterval); err != nil {
```

Comment:
Great catch, thank you for adding these in @jstr1121!

Reply:
I think on other modules these are missing - will need to add those on a new PR - since it's for stakeibc related PR.